### PR TITLE
Update NIRSpec TA Models

### PR DESCRIPTION
**Pull request by mfixstsci**
… for monitors as well.

### CODE REVIEW

File: jwql/website/apps/jwql/monitor_models/ta.py [hunk 1]
```diff
@@ -169,10 +168,10 @@ class NIRSpecMsataStats(models.Model):
     samroll = models.FloatField(blank=True, null=True)
     box_peak_value = ArrayField(models.FloatField())
     reference_star_mag = ArrayField(models.FloatField())
-    convergence_status = ArrayField(models.CharField())
+    convergence_status = ArrayField(models.CharField(max_length=100, blank=True, null=True))
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/monitor_models/ta.py]: I'm not sure how important it really is, but should we have a MAX_LEN_GENERIC_TEXT constant that holds these values of 100? I don't know how useful that would be. It's hard to predict how many characters you need for text fields that are descriptions or status values.

File: jwql/instrument_monitors/nirspec_monitors/ta_monitors/wata_monitor.py [hunk 2]
```diff
@@ -988,7 +989,7 @@ def update_ta_success_txtfile(self):
 
     def add_timezone(self, date_str):
         """Method to bypass timezone warning from Django"""
-        dt_timezone = datetime.fromisoformat(date_str).replace(tzinfo=timezone.utc)
+        dt_timezone = parser.parse(date_str).replace(tzinfo=timezone.utc)
```
Comment (bhilbert4) [inline jwql/instrument_monitors/nirspec_monitors/ta_monitors/wata_monitor.py:992]: Nirce. I'm glad there was a way to get this working.

**Review by bhilbert4 — APPROVED** [review]
LGTM!

File: jwql/website/apps/jwql/monitor_models/ta.py (hunk 1 above)
Comment (mfixstsci) [reply jwql/website/apps/jwql/monitor_models/ta.py]: I actually like this idea. I can implement it really quick

File: jwql/instrument_monitors/nirspec_monitors/ta_monitors/wata_monitor.py (hunk 2 above)
Comment (mfixstsci) [reply jwql/instrument_monitors/nirspec_monitors/ta_monitors/wata_monitor.py:992]: Yeah, thankfully there is a built that is able to do what we need 👍

File: jwql/website/apps/jwql/monitor_models/ta.py (hunk 1 above)
Comment (mfixstsci) [reply jwql/website/apps/jwql/monitor_models/ta.py]: @bhilbert4 it's in

Comment (bhilbert4) [reply jwql/website/apps/jwql/monitor_models/ta.py]: > @bhilbert4 it's in

Nice!

**Pull request merged**